Data/ByteString/Streaming/Char8.hs

```diff
@@ -676,30 +652,36 @@ newline :: Word8
 newline = 10
 {-# INLINE newline #-}
 
+-- | Promote a vanilla `String` into a stream.
 string :: String -> ByteString m ()
 string = chunk . B.pack . Prelude.map B.c2w
 {-# INLINE string #-}
 
-
+-- | Returns the number of times its argument appears in the `ByteString`.
```

Comment:
The `Char` is truncated to 8 bits.

Reply:
Sneaky, I was wondering how non-ASCII is handled here.